### PR TITLE
Add repos for `lein release`

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

**Comment by otfrom** [comment]
Build fail?

**Comment by acron0** [comment]
Arbitrary fails....

**Comment by acron0** [comment]
It's passed 3-4 times now so going to remove logging. @mantree I think the `batch-timeout` was kicking in and wrecking the merge code. Need to have a chat about this.

File: test/witan/workspace/acceptance/asfr_model.clj [hunk 1]
```diff
@@ -41,7 +41,7 @@
                      [:scaling-factors :historic-fertility]
                      [:historic-fertility :out]])
 
-(def all-functions (set witan-workflow))
+(def all-functions (-> witan-workflow flatten set))
```
Comment (otfrom) [inline test/witan/workspace/acceptance/asfr_model.clj:44]: flatten always worries me. mapcat?

**Comment by otfrom** [comment]
:shipit:

File: test/witan/workspace/acceptance/asfr_model.clj (hunk 1 above)
Comment (acron0) [inline test/witan/workspace/acceptance/asfr_model.clj:44]: I know what you're saying but this is only used for isomorphic fiddling, not actually in a code path (even a test).

**Pull request merged**